R/theme.R

```diff
@@ -0,0 +1,64 @@
+#' A ggplot2 theme for IHEID style plots
+#' 
+#' @importFrom ggplot2 theme
+#' @examples 
+#' \dontrun{
+#' ggplot(iris, aes(x = Sepal.Length, y = Sepal.Width)) + 
+#' geom_point() + ggtitle("Example Plot") + theme_iheid()
+#' }
+#' @export
+theme_iheid <- function(){
+  font <- "Helvetica"
```

Comment:
Or Helvetica Neue?

Reply:
Helvetica and Helvetica Neue are fonts that are available natively only on macOS and their equivalent is Arial on Windows machines. I'll try to find a workaround to set the font depending on the OS the user is running otherwise we might have to settle for a different font available across all platforms.

Reply:
I would recommend leaving it at "sans", the default option which works across platforms and works with base R. Users who would want to customize the fonts (and use Comic Sans MS for example) can on their own add other Windows and MacOS fonts by using the `{extrafont}`. More details [here](https://cran.r-project.org/web/packages/extrafont/README.html).

Reply:
Changed Helvetica to "sans" for the sake of interoperability. I will mention the option to add local fonts with`{extrafonts}` in the upcoming vignette.